test/Libraries/Microsoft.Extensions.Compliance.Redaction.Tests/Microsoft.Extensions.Compliance.Redaction.Tests.csproj

```diff
@@ -7,6 +7,11 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\src\Libraries\Microsoft.Extensions.Compliance.Redaction\Microsoft.Extensions.Compliance.Redaction.csproj" ProjectUnderTest="true" />
     <ProjectReference Include="..\..\..\src\Libraries\Microsoft.Extensions.Compliance.Testing\Microsoft.Extensions.Compliance.Testing.csproj" />
-    <ProjectReference Include="..\..\..\src\Libraries\Microsoft.Extensions.Hosting.Testing\Microsoft.Extensions.Hosting.Testing.csproj" />
+  </ItemGroup>
```

Comment:
Not sure I'm following. Where these dependencies transitively obtained from Hosting.Testing?

Reply:
Yep